types/node/fs/promises.d.ts

```diff
@@ -960,7 +965,7 @@ declare module 'fs/promises' {
      * If `persistent` is not supplied, the default of `true` is used.
      * If `recursive` is not supplied, the default of `false` is used.
      */
-    function watch(filename: PathLike, options?: WatchOptions | BufferEncoding): AsyncIterable<string>;
+    function watch(filename: PathLike, options?: WatchOptions | BufferEncoding): AsyncIterable<FileChangeInfo>;
```

Comment:
Actually yea, this should be `AsyncIterable<FileChangeInfo<string>>`

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55033/commits/5542877d670cf9a493899b5dca25573ff545c703